deployment/arm/deploy-region.sh

```diff
@@ -37,3 +39,12 @@ az group deployment create \
   --resource-group $resourceGroupName \
   --template-file region.json \
   --parameters location=$location apimName=$apimName appServicePlanName=$appServicePlanName frontendAppName=$frontendAppName backendAppName=$backendAppName appInsightsName=$appInsightsName
+
+cosmosEndPoint=`az group deployment show -g  rg-$globalScope -n Urlist-global-cosmosdb --query properties.outputs.documentEndpoint.value -o tsv`
+cosmosKey=`az group deployment show -g  rg-$globalScope -n Urlist-global-cosmosdb --query properties.outputs.accountKey.value -o tsv`
+
+echo "Configuring App service settings for $backendAppName"
+az webapp config appsettings set \
+-g $resourceGroupName \
```

Comment:
Indent arguments to be consistent with above.

Reply:
Done